ct_core/include/ct/core/Systems

```diff
@@ -31,6 +31,7 @@ Licensed under the BSD-2 license (see LICENSE file in main directory)
 #include "systems/discrete_time/SystemDiscretizer.h"
 
 #include "systems/discrete_time/linear/DiscreteLinearSystem.h"
+#include "systems/discrete_time/linear/DiscreteLTISystem.h"
```

Comment:
I added the new class to the convenience includes.
Amazingly, I was allowed to push directly to your fork @kvmanohar22  ;)